lightning-background-processor/src/lib.rs

```diff
@@ -415,7 +414,7 @@ where
 	let mut should_continue = true;
 	define_run_body!(persister, event_handler, chain_monitor, channel_manager,
 		gossip_sync, peer_manager, logger, scorer, should_continue, {
-			select! {
+			select_biased! {
```

Comment:
Would be good to have the reason for this change in the commit message or in a separate commit if unrelated.

Reply:
Added more context in commit message with https://github.com/lightningdevkit/rust-lightning/pull/1744/commits/0038a3f7dea705823939909c60bbd3826362bc1f. Let me know if it's sufficient.